components/Recipe/RecipeHead.tsx

```diff
@@ -0,0 +1,24 @@
+import Image from 'next/image';
+import { RecipeMeta } from '../../lib/types';
+import { Category, Clock, Root, Time, Title } from './RecipeHead.styles';
+
+export type RecipeHeadProps = {
+  meta: RecipeMeta;
+};
+
+export const RecipeHead = ({ meta }: RecipeHeadProps) => {
+  const { category, title, time, image } = meta;
+  const src = require(`../../_recipes/images/${image}`);
```

Comment:
Nit: This way webpack will bundle all image paths to the JavScript bundle. Its ok then we have 4 images, but if we have 1000 images it will be quite a large chunk of JavaScript.